db/migrate/20150713143149_remove_feedback_email_from_site.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveFeedbackEmailFromSite < ActiveRecord::Migration
```

Comment:
As there is a possibility we'll need to revert this, should we have an up/down on migrations? @pixeltrix